gui/biomes.lua

```diff
@@ -0,0 +1,213 @@
+-- Show biomes on the map.
+
+local gui = require('gui')
+local widgets = require('gui.widgets')
+local guidm = require('gui.dwarfmode')
+
+local biomeTypeNames = {
```

Comment:
@ab9rf is there any way to get these strings by lookup?

Reply:
not at present. someone could go into `df.map.xml`, add an `enum-attr` to the enum def,  and then add `item-attr` values to each `enum-item` so as to create one that could then be used by downstream tools, but nobody has ever done this